src/main/java/com/example/Restaurant_Management_System_REST_API/rest/controller/MenuRecordController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.Restaurant_Management_System_REST_API.rest.controller;
+
+import com.example.Restaurant_Management_System_REST_API.DTO.MenuRecordDTORequest;
+import com.example.Restaurant_Management_System_REST_API.DTO.MenuRecordDTOResponse;
+import com.example.Restaurant_Management_System_REST_API.exception.NotFoundInDatabaseException;
+import com.example.Restaurant_Management_System_REST_API.service.MenuRecordService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/menu/record")
+@AllArgsConstructor
+public class MenuRecordController {
+
+    private MenuRecordService menuRecordService;
```

Comment:
```suggestion
    private final MenuRecordService menuRecordService;
```

Are there any objections to this field being final? If not, I think it should be final.

Reply:
you are absolutely right!